src/test/resources/data/InstructorEditInstructorFeedbackPageTest.json

```diff
@@ -0,0 +1,465 @@
+            "institute":"TEAMMATES Test Institute 1"
+        },
+        "IEIFPTCoursehelper1":{
+            "googleId":"IEIFPTCoursehelper1",
+            "name":"IEIFPTCoursehelper1",
+            "isInstructor":true,
+            "email":"IEIFPTCoursehelper1@course1.tmt",
+            "institute":"TEAMMATES Test Institute 1"
+        },
+        "IEIFPTCoursehelper2":{
+            "googleId":"IEIFPTCoursehelper2",
+            "name":"IEIFPTCoursehelper2",
+            "isInstructor":true,
+            "email":"IEIFPTCoursehelper2@course1.tmt",
+            "institute":"TEAMMATES Test Institute 1"
+        },
+        "IEIFPTCoursehelper3":{
+            "googleId":"IEIFPTCoursehelper3",
+            "name":"IEIFPTCoursehelper3",
+            "isInstructor":true,
+            "email":"IEIFPTCoursehelper3@course1.tmt",
+            "institute":"TEAMMATES Test Institute 1"
+        },
+        "student1InIEIFPTCourse":{
+            "googleId":"student1InIEIFPTCourse",
+            "name":"Student 1 in course IEIFPTCourse",
+            "isInstructor":false,
+            "email":"student1InIEIFPTCourse@gmail.tmt",
+            "institute":"TEAMMATES Test Institute 1",
+            "studentProfile":{
+                "googleId":"student1InIEIFPTCourse",
+                "shortName":"Stud1",
+                "email":"i.m.stud1@gmail.tmt",
+                "institute":"TEAMMATES Test Institute 3",
+                "nationality":"U.S.A",
+                "gender":"male",
+                "moreInfo":"I am just a student :P",
+                "pictureKey":"asdf34&hfn3!@"
+            }
+        }, 
+        "student2InIEIFPTCourse":{
+            "googleId":"student2InIEIFPTCourse",
+            "name":"Student 2 in IEIFPTCourse",
+            "isInstructor":false,
+            "email":"student2InIEIFPTCourse@gmail.tmt",
+            "institute":"TEAMMATES Test Institute 1"
+        }
+        
+    },
+    "courses":{
+        "IEIFPTCourse":{
+            "id":"IEIFPTCourse",
+            "name":"IEIFPTCourse"
+        }
+    },
+    "instructors":{
+        "IEIFPTCourseinstr":{
+            "googleId":"IEIFPTCourseinstr",
+            "courseId":"IEIFPTCourse",
+            "name":"IEIFPTCourseinstr",
+            "email":"IEIFPTCourseintr@gmail.tmt",
+            "role":"Co-owner",
+            "isInstructor":true,
+            "displayedName":"Instructor",
+            "instructorPrivilegesAsText": "{\n  \"courseLevel\": {\n    \"canviewstudentinsection\": true,\n    \"cangivecommentinsection\": true,\n    \"cansubmitsessioninsection\": true,\n    \"canmodifysessioncommentinsection\": true,\n    \"canmodifycommentinsection\": true,\n    \"canmodifycourse\": true,\n    \"canviewsessioninsection\": true,\n    \"canmodifysession\": true,\n    \"canviewcommentinsection\": true,\n    \"canmodifystudent\": true,\n    \"canmodifyinstructor\": true\n  },\n  \"sectionLevel\": {},\n  \"sessionLevel\": {}\n}"
```

Comment:
Didn't Yuxuan merged a PR that affects this field recently? I think it is no longer a string. In any case, the line is too long.